<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gpkg2ShpDialog</class>
 <widget class="QDialog" name="Gpkg2ShpDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GPKG ‚Üí SHP</string>
  </property>
  <layout class="QHBoxLayout" name="mainLayout">

   <!-- Columna izquierda: opciones -->
   <item>
    <layout class="QVBoxLayout" name="leftLayout">

     <!-- Carpeta de entrada -->
     <item>
      <widget class="QLabel" name="inputLabel">
       <property name="text">
        <string>üìÇ Carpeta de entrada (GeoPackages):</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="inputLayout">
       <item>
        <widget class="QLineEdit" name="inputFolderLineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="inputBrowseButton">
         <property name="text">
          <string>Explorar...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>

     <!-- Carpeta de salida -->
     <item>
      <widget class="QLabel" name="outputLabel">
       <property name="text">
        <string>üíæ Carpeta de salida (Shapefiles):</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="outputLayout">
       <item>
        <widget class="QLineEdit" name="outputFolderLineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="outputBrowseButton">
         <property name="text">
          <string>Explorar...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>

     <!-- EPSG destino -->
     <item>
      <widget class="QLabel" name="epsgLabelTitle">
       <property name="text">
        <string>üåç EPSG (opcional):</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="epsgLayout">
       <item>
        <widget class="QLineEdit" name="epsgLineEdit"/>
       </item>
      </layout>
     </item>

     <!-- Log de ejecuci√≥n -->
     <item>
      <widget class="QLabel" name="logLabel">
       <property name="text">
        <string>üìú Registro de ejecuci√≥n:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="logTextEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Mensajes de ejecuci√≥n...</string>
       </property>
      </widget>
     </item>

     <!-- Botones -->
     <item>
      <layout class="QHBoxLayout" name="buttonsLayout">
       <item>
        <widget class="QPushButton" name="runButton">
         <property name="text">
          <string>‚ñ∂ Ejecutar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>‚ùå Cancelar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>

    </layout>
   </item>

   <!-- Columna derecha: descripci√≥n + advertencias -->
   <item>
    <layout class="QVBoxLayout" name="rightLayout">
     <item>
      <widget class="QTextEdit" name="descriptionText">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="html">
        <string>
         <![CDATA[
         <h1><b>gpkg2shp</b></h1>
         <p><b>Descripci√≥n de la herramienta:</b></p>
         <p>Convierte todos los archivos <b>GeoPackage (.gpkg)</b> de la carpeta seleccionada (y de sus subcarpetas) en <b>Shapefiles (.shp)</b>, 
            generando un shapefile por cada capa contenida en los GPKG.</p>
         <ul>
           <li><b>Carpeta de entrada:</b> donde est√°n los archivos GPKG.</li>
           <li><b>Carpeta de salida:</b> destino de los shapefiles exportados.</li>
           <li><b>EPSG (opcional):</b> reproyecta los datos al EPSG indicado.</li>
         </ul>
         <p><b>‚ö† Advertencias:</b></p>
         <ul>
           <li>Si un shapefile ya existe, ser√° sobrescrito.</li>
           <li>Si no se ingresa un EPSG o se ingresan caracteres no numericos las capas conservar√°n su CRS original.</li>
           <li>Si un EPSG es inv√°lido, la conversi√≥n fallar√°.</li>
         </ul>
         ]]>
        </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>

  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
